packages/react-core/src/core/types/wallet.ts

```diff
@@ -66,4 +69,31 @@ export type ConnectUIProps = {
    * theme of the connect wallet modal
    */
   theme: "dark" | "light";
+
+  /**
+   * `ConfiguredWallet` object of your wallet
+   *
+   * you can use this to connect to your wallet
+   *
+   * ### 1. Using `useConnect` hook
+   * ```ts
+   *  const connect = useConnect(configuredWallet);
+   *
+   *  connect(options);
```

Comment:
what is `options` in this case? Should we say `walletConfig` ?

Reply:
Yeah, it's wrong - updating it